### PR TITLE
feat: removed auth guard in get project details

### LINKED ISSUE #261: [FEATURE] Remove auth from the view project details page
### Remove Auth From the View Project Details Page.

Without login, the user should be able to view the project details.

### PR DESCRIPTION
**Pull request by rohitdasu**
### Fixes Issue

closes #261 

### Changes proposed

removed auth guard in get project details from FE and BE

### CODE REVIEW

File: components/Layouts/SharedLayout.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const SharedLayout: React.FC<IProps> = ({
             target="_blank"
             rel="noreferrer"
           >
-            MIT License
+            IT License
```
Comment (leewan-09) [inline components/Layouts/SharedLayout.tsx:39]: why?

Comment (rohitdasu) [reply components/Layouts/SharedLayout.tsx:39]: oops.. my bad

**Review by leewan-09 — APPROVED** [review]
LGTM

**Review by JakubChorzepa — APPROVED** [review]
LGTM :+1:

**Pull request merged**